apps/condo/domains/property/gql.js

```diff
@@ -1,8 +1,29 @@
-const { genTestGQLUtils } = require('@core/keystone/gen.gql.utils')
+/**
+ * Generated by `createschema property.Property 'organization:Text; name:Text; address:Text; addressMeta:Json; type:Select:building,village; map?:Json'`
+ * In most cases you should not change it by hands
+ * Please, don't remove `AUTOGENERATE MARKER`s
+ */
 
-const PROPERTY_FIELDS = '{ id dv sender organization { id name } name type address addressMeta map v deletedAt newId createdBy { id name } updatedBy { id name } createdAt updatedAt }'
-const Property = genTestGQLUtils('Property', PROPERTY_FIELDS)
+const { generateGqlQueries } = require('@condo/domains/common/utils/codegeneration/generate.gql')
+const gql = require('graphql-tag')
+
+const COMMON_FIELDS = 'id dv sender v deletedAt organization { id name} newId createdBy { id name } updatedBy { id name } createdAt updatedAt'
+
+const PROPERTY_FIELDS = `{ organization { id name} name address addressMeta type ticketsInWork flatsCount map ${COMMON_FIELDS} }`
+const Property = generateGqlQueries('Property', PROPERTY_FIELDS)
+
+const GET_TICKET_INWORK_COUNT_BY_PROPERTY_ID_QUERY = gql`
```

Comment:
We can use default `Ticket.useObjects()` with filter params.
see: apps/condo/pages/ticket/index.tsx:47

Reply:
This query is used in virtual field to get count, so I can't use useObjects here